src/main/scala/kamon/graphite/UDPBasedGraphiteMetricsSender.scala

```diff
@@ -26,102 +29,134 @@ class UDPBasedGraphiteMetricsSender(graphiteConfig: Config, metricKeyGenerator:
 
   import collection.JavaConverters._
 
-  val graphiteHost = graphiteConfig.getString("hostname")
-  val graphitePort = graphiteConfig.getInt("port")
-  val percentiles: Seq[Integer] = Some(graphiteConfig.getIntList("percentiles")).map(_.asScala).getOrElse(Nil)
+  private val graphiteHost = graphiteConfig.getString("hostname")
+  private val graphitePort = graphiteConfig.getInt("port")
+  private val percentiles: Seq[Integer] = Some(graphiteConfig.getIntList("percentiles")).map(_.asScala).getOrElse(Nil)
+
+  private val plaintextEncoder = new GraphitePlaintextEncoder()
```

Comment:
Good point. Could make configurable. Since we don't need the pickler at the moment didn't add. Changing this to be specified by configuration would be way to go though.